### PR TITLE
Using binding to PSD estimation

### PR DESCRIPTION
**Pull request by DStrelak**
partly addressing https://github.com/I2PC/xmipp/issues/132
deprecating:
computePSD()
composePSD()
computePSDs()

and adding new alternatives, which do not require repeated data store/load

### CODE REVIEW

File: pyworkflow/em/protocol/protocol_align_movies.py [hunk 1]
```diff
@@ -552,13 +551,41 @@ def averageMovie(self, movie, inputFn, outputMicFn, binFactor=1, roi=None,
         self.__runXmippProgram('xmipp_movie_alignment_correlation', args)
 
     def computePSD(self, inputMic, oroot, dim=400, overlap=0.7):
-        args = '--micrograph "%s" --oroot %s ' % (inputMic, oroot)
-        args += '--dont_estimate_ctf --pieceDim %d --overlap %f' % (dim, overlap)
+        import warnings
+        warnings.warn("Use psd = image.computePSD(overlap=0.4, xdim=384, ydim=384, fftthreads=1) instead",
```
Comment (azazellochg) [inline pyworkflow/em/protocol/protocol_align_movies.py]: can we set this options by default then?

Comment (DStrelak) [reply pyworkflow/em/protocol/protocol_align_movies.py]: Which option do you refere to?

**Comment by azazellochg** [comment]
(overlap=0.4, xdim=384, ydim=384, fftthreads=1

**Comment by DStrelak** [comment]
> (overlap=0.4, xdim=384, ydim=384, fftthreads=1

See https://github.com/I2PC/xmipp/blob/d9ecf72353e39aacd5596b9a8d246ba23469d4d5/bindings/python/python_image.cpp#L1165
I did not find a better way though.

File: pyworkflow/em/protocol/protocol_align_movies.py (hunk 1 above)
Comment (pconesa) [reply pyworkflow/em/protocol/protocol_align_movies.py]: @DStrelak can newones be called from inside the old ones?

Comment (pconesa) [reply pyworkflow/em/protocol/protocol_align_movies.py]: Or they are not compatible?

Comment (DStrelak) [reply pyworkflow/em/protocol/protocol_align_movies.py]: @pconesa Please see the latest commit

**Comment by pconesa** [comment]
@DStrelak could you resolve the conflicts, please. I'll review it after.

**Pull request merged**